test/local/convertToX96HexPrice.test.ts

```diff
@@ -0,0 +1,47 @@
+import { convertToX96HexPrice } from '@brink-sdk/internal';
+import { expect } from 'chai';
+
+describe('Utils', () => {
```

Comment:
probably could remove this top level describe

Reply:
done